ui/ui.go

```diff
@@ -133,6 +149,14 @@ func (h *prdHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 // Renders a templated asset in prd-mode. This strategy will embed
 // the sources directly in a script tag on the templated page.
 func renderForPrd(w io.Writer, c *content, cfgJson string, r *http.Request) error {
+	// If the requested asset is of type xml or text, short cut the execution
+	// using text/template package.
+	// See - https://github.com/etsy/hound/issues/239
+	if c.tplType == "xml" || c.tplType == "text" {
+		return c.ttpl.Execute(w, map[string]interface{}{
+			"Host": r.Host,
```

Comment:
Is there any reason not to pass the same data map to all templates? If not, seems like consistency would help us avoid surprises in future additions.

Reply:
No reason as such. However, you're right. Inconsistency may hurt in future additions. I'll fix that.